tests/suite/put/put-multi-5.yaml

```diff
@@ -0,0 +1,13 @@
+# Exercises the bug in zq issue 1152
```

Comment:
Let's give this file a more meaningful name. Maybe put-overwrite-c-a-b.yaml, put-overwrite-3rd-1st-2nd.yaml, or put-overwrite-last-field-first.yaml?

Reply:
I actually think that ztest should handle multiple zql tests in one yaml file and all of these put-multi-* tests should be in one file...

Reply:
ok